src/EditorFeatures/VisualBasic/LineCommit/CommitFormatter.vb

```diff
@@ -32,8 +31,8 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.LineCommit
 
         <ImportingConstructor>
         <SuppressMessage("RoslynDiagnosticsReliability", "RS0033:Importing constructor should be [Obsolete]", Justification:="Used in test code: https://github.com/dotnet/roslyn/issues/42814")>
-        Public Sub New(indentationManagerService As IIndentationManagerService)
-            _indentationManagerService = indentationManagerService
+        Public Sub New(inferredIndentationService As IInferredIndentationService)
```

Comment:
💡 If we have to update this constructor anyway, it would be good if we could eliminate the RS0033 suppression.